src/app/hierarchical-grid/hierarchical-grid-filtering-template/hierarchical-grid-filtering-template.component.html

```diff
@@ -57,32 +57,14 @@
 </ng-template>
 <ng-template #dateFilterTemplate igxFilterCellTemplate let-column="column" let-grid="column.grid">
     <div class="filter-cell">
-        <igx-date-picker [value]="getFilterValue(column)"
-                         (valueChange)="onDateSelected($event, column, grid)"
-                         [overlaySettings]="overlaySettings" mode="dialog">
-            <label igxLabel>Date</label>
-
-<!-- igxDatePickerTemplate has been removed.
-Label, prefix, suffix and hint can now be projected directly.
-See https://www.infragistics.com/products/ignite-ui-angular/angular/components/date-picker -->
-<ng-template igxDatePickerTemplate let-openDialog="openDialog" let-displayData="displayData">
-                <igx-input-group type="box" [displayDensity]="'compact'">
-                    <igx-prefix>
-                        <igx-icon>search</igx-icon>
-                    </igx-prefix>
-                    <input #input
-                        igxInput
-                        (click)="openDatePicker(openDialog)"
-                        placeholder="Filter..."
-                        tabindex="0"
-                        autocomplete="off"
-                        [value]="displayData | date"
-                        [readonly]="true"/>
-                    <igx-suffix *ngIf="input.value || input.value === 0" (click)="clearInput(column, grid)" tabindex="0">
-                        <igx-icon>clear</igx-icon>
-                    </igx-suffix>
-                </igx-input-group>
-            </ng-template>
+        <igx-date-picker #dp [value]="getFilterValue(column)" (valueChange)="onDateSelected($event, column, grid)" mode="dialog"
+            [displayDensity]="'compact'" placeholder="Filter..." type="box">
+            <igx-picker-toggle igxPrefix (click)="dp.open()">
```

Comment:
do not handle `click` in the `igx-picker-toggle` - this should be handled by date picker